scripts/crowbar-testbuild.py

```diff
@@ -48,25 +50,32 @@
 iosc = functools.partial(
     Command('/usr/bin/osc'), '-A', 'https://api.suse.de')
 
+orig_dir = os.getcwd()
+here = os.path.dirname(sys.argv[0])
```

Comment:
first of all, here is an incredibly broad term. And you're only using orig_dir joined together with here anyway, so it doesn't make sense to explode the code. rather do it in one place.